src/components/animate/DialogAnimate.tsx

```diff
@@ -0,0 +1,64 @@
+import { m, AnimatePresence } from 'framer-motion';
```

Comment:
You included a lot of functionality in relation to animation, but I believe you are currenlty not using the animation.

If we are sure we want to use `framer-motion` for animation, it would not hurt, but if not it would be a lot of unessary code.

Reply:
same, I am not sure witch components will require assets or animations so I would keep it for now